app/controllers/characters_controller.rb

```diff
@@ -19,6 +19,11 @@ def update
   
   def show; end
 
+  def destroy
+    @character.destroy
+    redirect_to characters_path
```

Comment:
It would be nice to add a flash message, check out https://github.com/Internship2017/star_wars_ffg/blob/master/app/controllers/characters_controller.rb#L29 for an example